client/src/components/NavLinkDropdownMenu/NavLinkDropdownMenu.js

```diff
@@ -0,0 +1,30 @@
+import React, {Component} from 'react';
+import "./NavLinkDropdownMenu.css";
+import { Redirect } from 'react-router-dom';
+
+class NavLinkDropdownMenu extends Component{
+  constructor(props){
+    super(props);
+    this.state = {
+      classList: "NavLinkDropdownMenu"
+    };
```

Comment:
Can you move this class into the `className` attribute in your JSX and convert this to a function component? It wont have any state